pkg/testing/ess/serverless_test.go

```diff
@@ -48,27 +47,23 @@ func TestStackProvisioner(t *testing.T) {
 	cfg := ProvisionerConfig{Region: "aws-eu-west-1", APIKey: key}
 	provClient, err := NewServerlessProvisioner(cfg)
 	require.NoError(t, err)
-	stacks := []runner.StackRequest{
-		{ID: "stack-test-one", Version: "8.9.0"},
-		{ID: "stack-test-two", Version: "8.9.0"},
-	}
+	request := runner.StackRequest{ID: "stack-test-one", Version: "8.9.0"}
```

Comment:
The second one was not redundant when the provisioner was used to deploy multiple at a time. Now that the code is changed it only does one at a time, so no need for the second stack. It would not be adding anything to the test.